echo-web/config/echo.yml

```diff
@@ -17,3 +17,5 @@ search:
   index: 'event_history'
 
 management.health.mail.enabled: false
+
+spring.freemarker.enabled: false
```

Comment:
this disables freemarker as a web view source (the `Configuration` object still gets instantiated)